OWNERS

```diff
@@ -0,0 +1,19 @@
+approvers:
+- calebevans
+- btjd
+- ascerra
+- myakove
+- rnetser
+options: {}
+reviewers:
+- madunn
```

Comment:
Did you check that all of those reviewers are RedhatQE org members?
Add reviewer that is not pat of the org will fail.

Reply:
I haven't but I will verify and invite them if they aren't members

Reply:
@myakove Many of them are not members but they need to be now that firewatch is moved to this org. I don't have access to invite them. Please either give me access or invite them to the org.

Reply:
For this PR we will be just limiting the reviewers list to us , and then add to it as more people start using.

Reply:
Removed other users